### ISSUE TITLE
Add support for clonning svn repository

### ISSUE THREAD

**Issue by mmorhun** [eclipse/che-theia#161]
### Description
Currently we support only git repositories in Factory plug-in which is responsible for cloning projects according to workspace configuration. But some projects use snv repsitories and it would be good to have support for them.

**Comment by sunix** [comment]
If we were to support svn, we should go through `git svn`.

**Comment by sunix** [comment]
> some projects

which projects ?

**Comment by mmorhun** [comment]
@sunix exactly, `git svn` is the way to go. I think implemetation shouldn't be hard.
As about projects, I do not understand your question. You want me to list to you projects which use svn?

**Comment by sunix** [comment]
I guess you are creating this issue because their is a customer needs or personal needs ? Just interested to know which ones.

**Comment by mmorhun** [comment]
There is no any customer requests so far, but I  think it might happen because still some projects, especially which were started long time before, use snv.
As about personal needs - I see no problem in bringing git svn in my tool container and clone needed projects. But it will be more comfortable to just specify it in devfile.
